mirror-service/src/main/java/org/hyades/MirrorServiceTopology.java

```diff
@@ -72,6 +81,18 @@ public Topology topology() {
                         .with(Serdes.String(), cyclonedxSerde)
                         .withName(processorNameProduce(KafkaTopic.NEW_VULNERABILITY, "nvd_vulnerability")));
 
+        // Vulnerability analyzers stream
+        // (K,V) to be consumed as (event ScanKey, ScanResult) from analyzers result topic
+        KStream<VulnerabilityScanKey, VulnerabilityScanResult> analyzerStream = streamsBuilder
+                .stream(KafkaTopic.VULN_ANALYSIS_RESULT.getName(), Consumed
+                        .with(scanKeySerde, scanResultSerde)
+                        .withName(processorNameConsume(KafkaTopic.VULN_ANALYSIS_RESULT)));
+        analyzerStream
+                .process(analyzerProcessorSupplier, Named.as("analyzers_vulnerabilities"))
```

Comment:
If the processor does not need access to state stores, or requires other kinds of dependencies, the dedicated `ProcessorSupplier` class can be replaced with a simple lambda:

```suggestion
                .process(AnalyzerProcessor::new , Named.as("analyzers_vulnerabilities"))
```

Reply:
Yes, good one, thanks I've pushed the change 👍🏼